resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/watson/ElementTree.java

```diff
@@ -733,14 +725,10 @@ public String toDebugString() {
 		return buffer.toString();
 	}
 
+	/** for debugging purposes only */
 	@Override
 	public String toString() {
-		ElementTree root = getParent();
-		while (root != null && root.getParent() != null) {
-			root = root.getParent();
-		}
-		return "ElementTree(" + treeStamp + "->" + (getParent() == null ? null : getParent().treeStamp) + "..." //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
-				+ (root == null ? null : root.treeStamp) + ")"; //$NON-NLS-1$
+		return "ElementTree(" + treeStamp + ")"; //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
Why don't we need parent timestamp now?

Reply:
(see screenshot) that tree information is now available in it's DeltaDataTree (ElementTree.tree)